inputremapper/gui/reader_service.py

```diff
@@ -145,7 +145,11 @@ def pkexec_reader_service():
         exit_code = os.system(cmd)
 
         if exit_code != 0:
-            raise Exception(f"Failed to pkexec the reader-service, code {exit_code}")
+            ex = Exception(f"Failed to pkexec the reader-service, code {exit_code}")
+            if ingore_errors:
+                logger.warn(ex)
```

Comment:
Some people might have very permissive setups by being member of the "device" user group or something. In that case, it might make sense to at least try to run `input-remapper-control --command start-reader-service{debug}` without pkexec as a regular user.

Reply:
Actually, I would like to move better pkexec handling to another PR and actually avoid having a need for a `-R` flag (or similar).

[Here](inputremapper/gui/reader_service.py) is what I would be looking for in a later change. But this fails many tests, so I guess the story is bigger than I thought.

Reply:
>  the "device" user group or something

I think it's called "input"

---

If the -R flag might be deprecated soon already, I wouldn't add it now

I think users being in the input group is the exception. I don't know right now how evdev behaves when an unprivileged user tries to use InputDevice objects and such. I guess you'd have to look out for any permission error and then exit. I would also use a special exit code for this >= 3.

Reply:
OK, I will remove the `-R` now but will leave an undocumented [env var](https://github.com/ubunatic/input-remapper/pull/2/files#diff-8d46828515cda696c8e9c4ad14726b07a5a01f518519ab8def95e5b237175236R139) for development. As said, let's discuss better privilege handling separately.